### PR TITLE
Add ability to pass custom certificate

### LINKED ISSUE #402: Self-signed certificate support
Hello,

I would like to submit a feature request related to our use of certificates for internal services. We utilize certificates issued by our internal, self-signed CA. Specifically, our Elasticsearch API use one of these certificates for the nodes within the cluster.

My request is to enable the capability of passing a certificate chain to the Yente client when it connects to Elasticsearch. I believe this enhancement should not be overly complex to implement. I am willing to contribute by preparing a patch myself. However, if someone could guide me on where to start, it would greatly expedite the process by reducing the time spent analyzing the client's workflow.

Additionally, some logs related to a failed connection

```
Cannot connect to ElasticSearch: 
TlsError("Cannot connect to host node1:9200 ssl:True 
[SSLCertVerificationError: (1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1006)')]", 
errors=(TlsError("Cannot connect to host node1:9200 ssl:True [SSLCertVerificationError: (1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1006)')]"
```

### PR DESCRIPTION
**Pull request by RMHogervorst**
Fix #402

Add the ability to pass the ca_cert arg to elastic search client

### CODE REVIEW

File: yente/settings.py [hunk 1]
```diff
@@ -148,6 +148,7 @@ def env_str(name: str, default: str) -> str:
 ES_PASSWORD = env_get("YENTE_ELASTICSEARCH_PASSWORD")
 ES_CLOUD_ID = env_get("YENTE_ELASTICSEARCH_CLOUD_ID")
 ES_SNIFF = as_bool(env_str("YENTE_ELASTICSEARCH_SNIFF", "false"))
+ES_CA_CERT = env_get("YENTE_ES_CERT_PATH")
```
Comment (pudo) [inline yente/settings.py:151]: I know this is a bit over, but perhaps the env var could be `YENTE_ELASTICSEARCH_CA_PATH`?

Comment (RMHogervorst) [reply yente/settings.py:151]: Yes! Way more consistant

**Comment by ZzeMR** [comment]
Hello !
Is this modification included in the latest docker image ??
This feature is real cool !

**Comment by pudo** [comment]
Yes! It's in 3.8.4. Just set `YENTE_ELASTICSEARCH_CA_PATH` to the CA certificate path.

**Pull request merged**